lib/std/array_list.zig

```diff
@@ -714,6 +714,9 @@ test "std.ArrayList.replaceRange" {
     try list_zero.replaceRange(1, 0, &new);
     try list_eq.replaceRange(1, 3, &new);
     try list_lt.replaceRange(1, 2, &new);
+
+    // after_range > new_items.len in function body
+    testing.expect(1 + 4 > new.len);
     try list_gt.replaceRange(1, 4, &new);
```

Comment:
@andrewrk see what I mean? this was already implicitly testing that `after_range > (new_)items.len` isn't resulting in out of bounds access. Just a confusing variable name in the function body. and again, sorry that I suck at github. maybe what we want is `assert(after_range < self.items.len)` at the top of the function body to communicate this invariant explicitly.